lib/accelerometer.js

```diff
@@ -242,16 +305,38 @@ function Accelerometer(opts) {
     this.toGravity = opts.toGravity || function(raw) { return raw; };
   }
 
+  if (!this.onEnable) {
+    this.onEnable = function(on) {};
```

Comment:
One step ahead of you on that one :)
On Sun, Jan 25, 2015 at 6:13 PM Rick Waldron <notifications@github.com>
wrote:

> In lib/accelerometer.js
> <https://github.com/rwaldron/johnny-five/pull/592#discussion_r23508724>:
>
> > @@ -242,16 +305,38 @@ function Accelerometer(opts) {
> >      this.toGravity = opts.toGravity || function(raw) { return raw; };
> >    }
> >
> > +  if (!this.onEnable) {
> > +    this.onEnable = function(on) {};
>
> Also, you can fake "enable"/"disable" for those that don't have hardware
> support by just returning from the dataHandler if state.enabled === false
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rwaldron/johnny-five/pull/592/files#r23508724>.
>